go.mod

```diff
@@ -40,6 +40,7 @@ require (
 	github.com/spf13/afero v1.6.0
 	github.com/spf13/cobra v1.0.0
 	github.com/spf13/viper v1.7.0
+	github.com/stretchr/testify v1.7.0
```

Comment:
Since we're using the [Golang testing package](https://github.com/ethersphere/bee/blob/master/CODINGSTYLE.md#testing), this dependency should not be included directly.